### ISSUE TITLE
CoBlocks: CoBlocks are not localized.

### ISSUE THREAD

**Issue by kwight** [Automattic/wp-calypso#37543]
We have not been loading language packs for our CoBlocks, leaving them un-localized for users.

<img width="1187" alt="Screen Shot 2019-11-04 at 1 09 27 PM" src="https://user-images.githubusercontent.com/349751/68722107-555bb580-0569-11ea-8487-19edd76965d2.png">

**Comment by davemart-in** [comment]
CoBlocks have moved over as part of the new Gutenberg Platform Team within the Create Group. cc @sirreal

**Comment by worldomonation** [comment]
Hi! I'm performing triage on older bugs as part of Quality Squad's efforts to cut down on backlog issue counts.

I took the following steps:
1. change the interface language to Japanese from `/me`.
2. start a post from one of my sites.
3. search for and insert a variety of CoBlocks.

For all blocks I have inserted they were localized to Japanese.

For additional testing, I then switched the interface language over to French and reopened the same post.
All of the helptext in the blocks were switched over to French as expected.
Interestingly certain blocks with default text __within__ the CoBlocks themselves eg. Hero > Title remained in Japanese. I will file that as a separate issue.

I propose that this issue be closed as the happy path for interacting with CoBlocks no longer exhibits this issue.

![Screen Shot 2021-03-18 at 4 01 27 PM](https://user-images.githubusercontent.com/6549265/111712272-85286680-880a-11eb-927c-678157fce19c.png)
![Screen Shot 2021-03-18 at 4 03 34 PM](https://user-images.githubusercontent.com/6549265/111712273-85286680-880a-11eb-9feb-ca50a6602570.png)
